setup.sh

```diff
@@ -1,76 +1,18 @@
 #!/bin/bash
 
-time=`TZ=Asia/Jerusalem date '+%Y%m%d_%H%M'`
-VAG_COMMON_DIR="/vagrant"
-HOME="/home/vagrant"
-LOG_DIR="$VAG_COMMON_DIR/log"
-OUTFILE="$LOG_DIR/${time}_vagrant-out.log"
-FLASK_APP_PY="$VAG_COMMON_DIR/flask_init.py"
-FLASK_EXEC="$HOME/runFlask.sh"
-FLASK_LOG="$LOG_DIR/flask-out.log"
-KILLFLASK_SCRIPT="$VAG_COMMON_DIR/killFlask.sh"
-FLASK_ONBOOT=false # Currently Flask On Startup feature disabled [ Not Supported ]
+echo "updating apt before installation"
+sudo apt update
 
-runSetup() {
-    step=1
+echo "installing python 3.7"
+sudo apt install -y python3.7 
 
-    if [ ! -d $LOG_DIR ]; then
-        mkdir $LOG_DIR
-    else
-        # Deleting all vagrant-out log files except for the 5 most recent
-        cd $LOG_DIR
-        ls -t | grep vagrant-out.log$ | tail -n +6 | xargs -d '\n' -r rm --
-        cd $HOME
-    fi
-    touch $OUTFILE
+echo "installing python3-pip"
+sudo apt install -y python3-pip
+pip3 install --upgrade pip
 
-    if [ ! -e $FLASK_LOG ]; then
-        touch $FLASK_LOG
-    fi
+echo "installing flask"
+#'python3 -m' has been added to avoid warnings
+python3 -m pip install flask
```

Comment:
what warnings are you getting? and are they bothering install?
calling pip3 and python3 -m pip _should_ give the same results.

Reply:
This is the warning:
**WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
    default: Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
    default: To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.**

It doesn't bother the installation, just shows this warning. When I add "python3 -m" before "pip" this warning doesn't appear. According to the issue above, it seems that some of the distributions need this extension to run "pip" without warnings. 
I tried to run older version of "pip", but then the VM suggested me to update to a newer version. When I upgraded the "pip", the warning above appeared.

Reply:
if you get rid of the line pip3 install --upgrade pip you shouldn't have this problem. 
what is happening is that it installed 2 different versions of pip on your machine.

Reply:
Do you recommend using older version of git instead of add "python3 -m" extension?

Reply:
I recommend either using your package manager (apt-get install) and use the version it gives you or downloading pip directly (https://pip.pypa.io/en/stable/installing/) and using the latest version. However, mixing the two causes problems.

Reply:
ack